Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerController : MonoBehaviour {
+
+  public float Speed;
+
+  private Rigidbody rigidbody;
+
+  void Start () 
+  {
+    rigidbody = GetComponent<Rigidbody> ();
+  }
+
+  void FixedUpdate () 
+  {
+    float moveHorizontal = Input.GetAxis ("Horizontal");
+    float moveVertical = Input.GetAxis ("Vertical");
```

Comment:
[MUST]
Do not use nonphysical process in FixedUpdate, because FixedUpdate will not update per frame but update in phiysic cycle whose FPS depend on the fixed step time.
https://docs.unity3d.com/Manual/ExecutionOrder.html